cluster/addons/rbac/node-proxy-role.yaml

```diff
@@ -10,15 +10,9 @@ rules:
   - ""
   resources:
   - nodes/proxy
-  verbs:
-  - create
-  - get
-- apiGroups:
-  - ""
-  resources:
   - nodes/log
   - nodes/stats
   - nodes/metrics
   - nodes/spec
```

Comment:
I can rename the role if that helps... this thing's job is to proxy to the kubelet API... to do that, it needs full access to the kubelet API.